pipelines/backend_tests.go

```diff
@@ -2,21 +2,49 @@ package pipelines
 
 import (
 	"context"
-
 	"dagger.io/dagger"
+	"fmt"
 	"github.com/grafana/grafana-build/containers"
+	"log"
+	"strings"
 )
 
+var IntegrationDatabases = []string{"sqlite", "mysql", "postgres"}
+
+// list of strings, string
+func Contains(arr []string, v string) bool {
```

Comment:
This probably needs a comment that starts with `contains`.

This function should also be unexported. In general, consider function identifiers to be the combination of the package they're in and the name of the function.

This function isn't just `Contains`, it's `pipelines.Contains`. Because it's exported, the implication is that it will be used outside of the pipelines package, and thus the full identifier doesn't imply that functionality.

Instead, make this `contains` so that it's known that it's only really used in this package.